window/src/lib.rs

```diff
@@ -71,6 +71,13 @@ pub enum MouseCursor {
     SizeLeftRight,
 }
 
+#[derive(Debug)]
```

Comment:
as suggested in a comment above, I think you can and should move this into the `config` crate.

```suggestion
#[derive(Debug, Copy, Clone, PartialEq, Eq)]
```